packages/generators/generators/src/plops/__tests__/content-type.test.ts

```diff
@@ -62,7 +62,7 @@ describe('Content Type Generator', () => {
       },
       attributes: {},
     });
-  });
+  }, 30000);
```

Comment:
urghhhh, we still have "unit" tests with file effects, I might add this to the tech debt db 😆

Reply:
yeah not gonna fix it in there we have enough changes already :D